### ISSUE TITLE
[Presentation] Sign-off needs adjusting

### ISSUE THREAD

**Issue by MicheleAWilliams-A11y** [w3c/wai-people-use-web-videos#55]
The sign-off is almost there but doesn't quite flow/connect:
> "All this has one thing in common: your design can include or exclude people."

Also wouldn't quite fit the other videos, particularly the user stories.

Perhaps rather than "your design" it's "how we design technology"? Also adding "of" to better complete the sentence.

A few iterations (things in brackets are options):

>All of this has one thing in common: how we design technology can include or exclude people.

>All of this has one thing in common: technology can be built to [either] include or exclude people [like me _- if user story_]. [_Optional:_ Which way are you going to go? OR Which type of technology will you create?]

>In the end, [there's 1 common point]: technology can *include* or *exclude* people. [What type of technology will you create?]

**Comment by bakkenb** [comment]
I like working to refine this sign-off / tag line a little more.
I like Michele's version...

> All of this has one thing in common: how we design technology can include or exclude people.

Maybe we can keep tweaking that a bit, or let the WG react to it in a survey question.

**Comment by eoncins** [comment]
Hello, I agree about the need of rephrasing the sign-off, what I miss is the explicit mention to the term "accessibility" which is at the beginning of the perspective videos "web accessibility: essential for some useful for all".  At the same time I do not know if including the term "technology" is a good option because to me accessibility is beyond the "technical" aspect. 
Some proposals, not really a big fan of them:
"All of this has one thing in common: accessible design, includes/cares for all people"
"All of this has one thing in common: accessible design, leaves no one behind"

**Comment by nitedog** [comment]
Relates to issue #12

**Comment by nitedog** [comment]
Some more brainstorms, trying to combine the approach of the Perspectives videos with the objectives of HPWDs:

- "Web accessibility: your design makes the difference"
- "Web accessibility: your design choices make the difference"
- "Web accessibility: your design choices are critical"
- "Web accessibility: how you design makes a difference"
- "Web accessibility: design for everyone, it's essential for some"
- "Accessible design: essential for some, useful for all"

**Comment by eoncins** [comment]
Thanks Shadi for the wrap up, 

Just a comment regarding the mention: "designers that understand my needs usually perform best". The problem is centring the attention on a single responsibility = "designers", which is not the same as "design". 

On the other side, "websites that consider me work better for me". Maybe rephrasing it into something like "websites that ask me about my accessibility needs work better". Just brainstorming, as you said.

**Comment by carlosapaduarte** [comment]
I'm not sure that including the word "design" is inclusive. I may not be fully aware of the target audience for these resources, but I assume that roles other than designers are included. Using "design" might make non-designers think the resources are not for them. That is why I prefer options that do not include it. My two favourite so far are:

* "Accessibility: essential for some, useful for all"
* "Web accessibility: your choices makes the/a difference"

**Comment by JenniferChadwick** [comment]
Thanks @MicheleAWilliams-A11y @nitedog and @carlosapaduarte for the brainstorming and suggestions. 

I am completing the EO survey on this and just adding my thoughts, as Carlos' favourites are mine as well. 

"Accessibility: essential for some, useful for all"
"Web accessibility: your choices makes the/a difference"

Thanks!
Jenn

**Comment by jade-mc** [comment]
Agree that 'design' might be problematic, as we're viewing it in quite a specific way, but the audience for the videos might have an alternative perspective, I think this came up in one of our meetings.

**Comment by lakeen** [comment]
brainstorm:
Web accessibility: how we design technology matters.

**Comment by sharronrush** [comment]
I get the point about 'design' being easily misunderstood as 'visual design.' 
another brainstorm:
Web accessibility: delightful user experiences for all

**Comment by bakkenb** [comment]
After reading the Auditory and the Presentation scripts again and thinking about the content and message that is conveyed throughout the scripts, I really like what Sharron is proposing here. Some videos will be about the person, some are about the disability and barriers, while others are about the tools and technology needed to make them work. It is all about making it work so the user has a delightful experience. This one gets my vote...

> "Web accessibility: delightful user experiences for all"

Another option could be: "Web accessibility: include and delight the user"

**Comment by AndrewArch** [comment]
Have to agree with @daniel-montalvo about "delightful" vs "effective/efficient" - pay tax is never going to be delightful, but I do want the process to be efficient :) I also agree with dropping the word "design" as it only reflects some roles in the overall process of creating an accessible experience.

**Comment by kheitm** [comment]
I also agree with the limits of using the term "design". Also, while some of our technology interactions are meant to generate delight, most of the day-to-day interactions are intended to be (hopefully) fast and efficient.

**Comment by iadawn** [comment]
It may not be in the sign off but having the user in the video talking to the audience about how their work impacts the users opportunities and independence is a strong message.

**Comment by sharronrush** [comment]
+1 to "impacts the users opportunities and independence"

**Comment by bakkenb** [comment]
A thought that came up during the video sub-group meeting this week...

Can we come up with a tag-line, or closing sign-off, that pushes the video audience to ask themselves "what is the next step for me?" or "now that I know this information, what should I be doing about it?"

Here are some ideas, but they may be too wordy and need some refinement or different approach:

> Understanding diverse abilities and how people access website and applications is the beginning, what is your next step?

> Understanding diverse abilities, ways people access digital content, and the barriers they face is a great start, now what are you going to do to advance accessibility?

> Help unleash the potential of the web for everyone. What can you do to support thoughtful and deliberate design that works for everyone?

**Comment by sharronrush** [comment]
Great thinking but yes a bit too wordy. How about:
Inclusion starts by understanding user experience - what are your next steps?
Now you know more about barriers faced by disabled users - what will you do next?
Design choices may create barriers for some - your choices can avoid them.

**Comment by kakinney** [comment]
I really liked the thoughts of the video sub-group, but Is it possible that the tag line/ end of the videos can incorporate wording to support the idea that this is the one perspective but there are also many other perspectives.  Everyone is so different in how the interact with information, and this is one perspective of the many out there.   

So along the same lines - "This is how you can help me.  How can you also help others? " type of question... 

> A thought that came up during the video sub-group meeting this week...
> 
> Can we come up with a tag-line, or closing sign-off, that pushes the video audience to ask themselves "what is the next step for me?" or "now that I know this information, what should I be doing about it?"
> 
> Here are some ideas, but they may be too wordy and need some refinement or different approach:
> 
> > Understanding diverse abilities and how people access website and applications is the beginning, what is your next step?
> 
> > Understanding diverse abilities, ways people access digital content, and the barriers they face is a great start, now what are you going to do to advance accessibility?
> 
> > Help unleash the potential of the web for everyone. What can you do to support thoughtful and deliberate design that works for everyone?

**Comment by shawna-slh** [comment]
Idea A:
For stories: **You** can help make accessible technology **enabling** for people like me.
For diverse abilities: **You** can help make accessible technology **enabling** for people with auditory disabilities.
For tools and techniques: **You** can help make accessible technology **enabling** for people with perceptual disabilities.

<hr>

Some thoughts on: “**You** can help make accessible technology **enabling** for people like me”:
1. “You can make…” is more direct; however, some people watching the videos will not be able to _directly_ make technology enabling. Including ‘help’ makes it relevant to everyone watching the video: “You can help make…”.
2. ‘accessible’ is not necessary in “… make accessible technology enabling…” &mdash; could be just “… make technology enabling…”; however, I think good to get ‘accessible’ or ‘accessibility’ in the tagline.
3. Thought about “… make technology **enabling** or **disabling** for people like me” &mdash; while I really like clarifying that design/development can be disabling or enabling, I think for a tagline, it might be better to keep it simple and positive? Also, ‘accessible technology’ wouldn’t work smoothly if you have ‘enabling or disabling’.
4. Is “enabling” too jargony or not well understood by some? Maybe “inclusive” is better, which is a buzz word? “You can help make accessible technology inclusive to people like me.”

5. “for people” or “to people” 
5.a. ‘enabling for people like me’ or  ‘enabling to people like me’?
5.b. if ‘inclusive’ rather than ‘enabling’ then ‘to’ &mdash; “…inclusive to people like me.”

6. I think we don’t need ‘web’ or ‘digital’. The statements work without it. And good to be broad and succinct.

7. “people like me” makes it more real and direct, and is in synch with COGA suggestion in https://github.com/w3c/wai-people-use-web-videos/issues/193  (COGA suggests using “me” or “people like me” in the tagline).

<hr>

Idea B:
For stories: **You** can help make accessible technology **inclusive** to people like me.
For diverse abilities: **You** can help make accessible technology **inclusive** to people with auditory disabilities.
For tools and techniques: **You** can help make accessible technology **inclusive** to people with perceptual disabilities.

7. Maybe move ‘accessible’:
7.a. “You can help make technology accessible and inclusive to people like me.” (‘and’ somehow makes it wimpier)
7.b. “**You can help make technology accessible so it’s inclusive to people like me.**”

**Comment by iadawn** [comment]
+1 for @shawna-slh general ideas.
-1 for 'enabling'. I don't know that as a word it is clear enough in this context.
+1 for 'inclusive' or, possibly 'include'.
-1 to 'accessible technology' - 'accessible technology' by definition should be inclusive. I think this would need to drop 'accessible' as the accessible stuff isn't the problem :)

A shorter alternative might be something like: 'Your work includes me'

**Comment by shawna-slh** [comment]
> For stories: You can help make accessible technology inclusive to people like me.
> For diverse abilities: You can help make accessible technology inclusive to people with auditory disabilities.
> For tools and techniques: You can help make accessible technology inclusive to people with perceptual disabilities.

Another thought: Use the same sign-off for all the videos! For diverse abilities and for tools and techniques videos, use clips from the stories -- maybe 2-4 clips for each video.

**Comment by bakkenb** [comment]
>Another thought: Use the same sign-off for all the videos! For diverse abilities and for tools and techniques videos, use clips from the stories -- maybe 2-4 clips for each video.

Expanding on this idea a bit...
When there are more than 2 or 3 protagonists in the video, the people who would be in these quick end clips could be those who's struggles really showed in the video, not the ones that are only demonstrating a tool or technique.

**Comment by shawna-slh** [comment]
discussion in [30 September EOWG meeting](https://www.w3.org/2022/09/30-eo-minutes.html#item02)

**Comment by MicheleAWilliams-A11y** [comment]
Per our subgroup meeting today (Oct 4, 2023) I'm thinking we might have different sign-offs for the different videos. The Diverse Abilities and Barriers videos, for instance, are trying to introduce _such_ a wide range of disabilities that it may be helpful to reinforce that in the closing. Not creating the full wording here but perhaps the ending says something like, "This is just an introduction, keep learning more." The other videos could also say that but their examples are a bit more bound. Just something to add to the discussion.

**Comment by nitedog** [comment]
Change suggested by EO has been addressed in the "disabilities" and "techniques" videos (currently being surveyed).

**Comment by shawna-slh** [comment]
I'm thinking that we want the closing  URI to be w3.org/WAI/people and we create a new landing page. I'm happy to take the lead on that.

**Comment by nitedog** [comment]
@shawna-slh please also check the text of the front plates, these are currently the document titles (which are sometimes quite long). Do we want to have the overall resource name "How People with Disabilities Use the Web" appear visually in addition to being narrated? Also, do we want to use the same fonts, colors, and style as in the Perspectives videos for the start and end of these videos?

**Comment by shawna-slh** [comment]
To have a clean list of what's needed, I created https://github.com/w3c/wai-people-use-web-videos/issues/286 -- and assigned it to Shawn & Kevin

Feel free to add additional input on this issue

**Comment by shawna-slh** [comment]
_below is just for the record._

“These are ways to make technology work for me.” doesn't work well for some of the scripts because the thing said before it is something that does *not* work:

* [Alan](https://wai-people-use-web-videos.netlify.app/people-use-web/videos/stories/alan/): “The most difficult thing is small buttons without labels – they are difficult to click through speech control, which disrupts my flow.” “These are ways to make technology work for me.”

* [Lee](https://wai-people-use-web-videos.netlify.app/people-use-web/videos/stories/lee/): “Another example is when charts don’t have symbols as well as the colors for lines and bars – I can’t identify them from color alone….” “These are ways to make technology work for me.”

* [Luis]( https://wai-people-use-web-videos.netlify.app/people-use-web/videos/stories/luis/): “Otherwise, it takes me even longer to find the words in the dictionary….” “These are ways to make technology work for me.”

<hr>

Kevin and I talked about alternatives.

One idea was "include me". However, I think does not work because there is too much "distance" between the persona and viewer.

**Comment by jade-mc** [comment]
If you want to keep everything nice and tidy with the same sign offs, then it might be worth just adding in something positive at the end/changing the last sentence a little: 
Suggestions: 

Alan: It makes such a difference when everything's bigger and labelled properly.
Lee: A lot of the suppliers we work with are sometimes really bad at doing this [sigh] but I'm really glad that my team are really careful with colour and work hard to make sure it's accesible.
Luis: It takes me ages to find words in the dictionary, so it really helps when coaches use words I already know.

**Comment by nitedog** [comment]
Agree with @jade-mc. @iadawn, if there is still opportunity for minimal text change (doesn't change props, location, etc.), please consider:

- Alan: "The most difficult thing is small buttons without labels – they are difficult to click through speech control, which disrupts my flow." -> "The most difficult thing is small buttons without labels, like these binoculars – I'm not sure if the right command is "search" or "go" or something else. When buttons have a name, I can simply say that without disrupting my flow."
- Lee (now Lexie): "Another example is when charts don’t have symbols as well as the colors for lines and bars – I can’t identify them from color alone. My team is usually careful about this but sometimes the suppliers we work with aren’t [sigh]." -> "It's also very important that charts have symbols in addition to colors for the lines and bars, rather than relying on color alone."
- Luis (now Sophie): "It helps me when coaches use words I already know. Otherwise, it takes me even longer to find the words in the dictionary. One day I’ll make it!" -> "I also use a dictionary to find words that I don't know, and it takes me longer to read text with difficult words. It helps me when coaches use words I already know, and most do. One day I'll make it!"

**Comment by iadawn** [comment]
@jade-mc, @nitedog thanks both. I think it would be good to add those in but I don't think we have time to create a suitable line for all nine characters, review them and add them to the scripts. The simple, conmsistent hand off that we have come up with should work for all instances. If there is any significant concern with that then I can take a look.

Just to keep it all here, the final line that we are using is:

> "You can help make technology accessible to me."

This will be the same for all characters.

**Comment by jade-mc** [comment]
I was trying to suggest adding text in/amending it slightly so we would be able to keep the 9 endings the same as they are and address Shawn's comments? (Have we got crossed wires somewhere?)
